### ISSUE TITLE
when the HS can't be reached on login, maybe suggest checking if it is being blocked by an ad blocker

### ISSUE THREAD

**Issue by uhoreg** [vector-im/riot-web#3978]
in addition to the suggestion to 'check if your ssl is trusted'

**Comment by uhoreg** [comment]
Obviously, checking for an ad blocker would only apply to web, and not the desktop app.

**Comment by lampholder** [comment]
I'm guessing you've experienced its being blocked by an adblocker? It would be interesting to know if there were anything we could do to stop traffic being misidentified as something the adblocker needs to filter.

Also I would love to see the SSL check hint, since that bites me every three months :P

**Comment by uhoreg** [comment]
FTR, it wasn't me personally who experienced it.  It was someone else in one of the rooms.  (I forget who, or which room it was in.)

**Comment by uhoreg** [comment]
closing in favour of #4247
